frontend/services/entities/hosts.ts

```diff
@@ -310,12 +310,11 @@ export default {
 
     return sendRequest("POST", path);
   },
-  search: (searchText: string) => {
-    const { HOSTS } = endpoints;
-    const path = `${HOSTS}?query=${searchText}`;
-
-    return sendRequest("GET", path);
-  },
+  // search: (searchText: string) => {
+  //   const { HOSTS } = endpoints;
+  //   const path = `${HOSTS}?query=${searchText}`;
+  //   return sendRequest("GET", path);
+  // },
```

Comment:
Host search API doesn't seem to be used by the frontend. It uses list hosts instead. Commenting out this code for now and we can delete it later assuming we don't see QAWolf e2e test failures.

Reply:
is this included as part of the back-end PR so you can revert the change as part of the UI?